bin/ghost

```diff
@@ -4,6 +4,6 @@
 // provide a title to the process
 process.title = 'ghost';
 
+require('../lib/update_notifier');
```

Comment:
This is being a bit nitpicky, but could you move the `update_notifier` into the `utils` folder/make it `update-notifier` instead? That would keep it a bit more consistent with the file naming schema 😄